File: reactjs-client/src/common/ReportingAuthority.js
const ReportingAuthority = (props) => {
    const {authority} = props;
    const listItems = authority.agenda.map((number) =>
    <li>{number}</li>
    );
    return(
        <div>
            <p><label>Name:</label>{ authority.name }</p>
            <p><label>Objective:</label>{ authority.objective}</p>
            <p><label>Agenda:</label><div><ul>{listItems}</ul></div></p>
        </div>
    );
}

export default ReportingAuthority;

File: rest-server/src/main/java/org/mars_sim/rest/model/ReportingAuthority.java
package org.mars_sim.rest.model;

import java.util.List;


public class ReportingAuthority {
    private String name; 
    private String objective; 
    private List<String> agenda;
    
    ReportingAuthority(String name, String objective, List<String> agenda) {
        this.name = name;
        this.objective = objective;
        this.agenda = agenda;
    }
    public String getName() {
        return name;
    }
    public String getObjective() {
        return objective;
    }
    public List<String> getAgenda() {
        return agenda;
    }
    
    
}
